templates/web/base/report/_inspect.html

```diff
@@ -63,19 +64,38 @@
         [% END %]
 
         [% IF permissions.report_inspect %]
-        <p>
-          <label for="state">[% loc('State') %]</label>
-            [%# XXX this is duplicated from admin/report_edit.html, should be refactored %]
-          <select name="state" id="state" class="form-control">
-            [% FOREACH group IN state_groups %]
-              <optgroup label="[% group.0 %]">
-                [% FOREACH state IN group.1 %]
-                  <option [% 'selected ' IF state == problem.state %] value="[% state %]">[% state_pretty.$state %]</option>
-                [% END %]
-              </optgroup>
-            [% END %]
-          </select>
-        </p>
+          <p>
+            <label for="state">[% loc('State') %]</label>
+              [%# XXX this is duplicated from admin/report_edit.html, should be refactored %]
```

Comment:
I think this has been done in the 75-stateful-response-template branch, note for merging.